bdsf/functions.py

```diff
@@ -753,7 +753,7 @@ def deconv2(gaus_bm, gaus_c):
     return (bmaj, bmin, bpa), ifail
 
 
-def get_errors(img, p, stdav, bm_pix=None):
+def get_errors(img, p, stdav, bm_pix=None, fixed_to_beam=False):
```

Comment:
There's no documentation for the new `fixed_to_beam` argument. Also, you may want to make clear what kind of errors you're referring to exactly. I guess numerical errors, but that's not clear from the function name and the documentation. Also Condon 1997 may need a reference for the uninformed like me :).

Reply:
Good points -- I updated the docstring with more info.